client/src/Components/Menu.js

```diff
@@ -0,0 +1,91 @@
+import React, { useState, useContext, useEffect } from "react";
+import { CartContext } from "./CartContext";
+import { motion } from 'framer-motion'
+import axios from "axios";
+import Button from "react-bootstrap/Button";
+import Card from "react-bootstrap/Card";
+import brisket from "../Images/brisket.jpg";
+import pork from "../Images/Pork-Butt.jpg";
+import styles from "../Styles/Menu.module.css";
+axios.defaults.baseURL =
+  process.env.REACT_APP_baseURL || "http://localhost:4000";
+
+const Menu = () => {
+  const [cart, setCart] = useContext(CartContext);
+  const [menuItems, setMenuItems] = useState();
+  // use effect will retrieve items and their prices from the server to prevent price
+  // manipulation and set them in state variable
+  useEffect(() => {
+    const getMenuItems = async () => {
+      let data = await axios.get("/api/menu");
+      let menuArray = data.data.result
+      let menu = menuArray.map(menuItem => {
+        return {...menuItem, quantity: 0}
+      })
+      setMenuItems(menu);
+    };
+    getMenuItems();
+  }, []);
+  // on click will set the menu items in cart but only their _ids,
+  // only _ids will be sent to server then prices will be fetched from mongoDB
+  const handleClick = (item) => {
+    if (cart.length === 0){
+    setCart([item]);
+    }
+    let foundProduct = cart.find(cartItem => cartItem._id === item._id)
+    if (foundProduct) {
+      foundProduct.quantity++
```

Comment:
its good practice to not mutate state directly. you'll want to use `setCart` with the new quantity